app/helpers/common/priceFormatter.js

```diff
@@ -1,16 +1,22 @@
 export const formatPrice = (priceValue) => {
   const formattedPrice = priceValue.toLocaleString(undefined, {
-    minimumFractionDigits: 3, maximumFractionDigits: 3,
+    minimumFractionDigits: 2, maximumFractionDigits: 4,
   });
 
-  const truncatedTo2dp = formattedPrice.substring(0, formattedPrice.indexOf('.') + 3);
-
-  return truncatedTo2dp;
+  return formattedPrice;
 };
 
 export const formatDecimal = (priceValue) => {
```

Comment:
this function returns either a number or a string. id prob say it should be consistent, but not major if its the only change

Reply:
I think regarding the UI, it's ok to always return a string as we should not be doing any maths anyway and just displaying.